### ISSUE TITLE
Verify the OCS ID

### ISSUE THREAD

**Issue by nickvergessen** [owncloud/search_lucene#85]
The current entry of the ocsid is `166057` in all branches:
https://github.com/owncloud/search_lucene/blob/master/appinfo/info.xml#L44
https://github.com/owncloud/search_lucene/blob/stable8/appinfo/info.xml#L44
https://github.com/owncloud/search_lucene/blob/stable7/appinfo/info.xml#L34

However that app is only for oC 7: https://apps.owncloud.com/content/show.php?content=166057
oC 8.0 (and later**?**) should use `168709`: https://apps.owncloud.com/content/show.php?content=168709

This causes problems, when people try to enable the app from the recommended section rather then the disabled section: https://github.com/owncloud/core/issues/16092

Manually changing the ocsid resolves the issue.

**Comment by MorrisJobke** [comment]
> Manually changing the ocsid resolves the issue.

So we simply need to change this in stable8 and master to `168709`?

**Comment by nickvergessen** [comment]
well, master might need a new ocsid, im not familar with that.

**Comment by DeepDiver1975** [comment]
well - one app - one id????

The app should be listed in the appstore only once as well?

hmmm - I don't get the concept - sorry

**Comment by nickvergessen** [comment]
Well as far as I understand it, the app store can only contain one download for one ocsid. So we can not provide different versions depending on the requested oC version. Thats why "they" started to use multiple entries for each version.

**Comment by RealRancor** [comment]
Mhhh, that should be now fixed for quite some time. The appstore can now provide different downloads / versions for one single ocsid so it should be safe to use the new 168709 ocsid.

Another issue is, that the "All versions" download at https://apps.owncloud.com/content/show.php?content=168709 is still providing the old 166057 ocsid. Have raised https://github.com/owncloud/search_lucene/issues/113 here separately.
